src/components/confirmationPopup.js

```diff
@@ -0,0 +1,28 @@
+import Popup from "./Popup";
+
+class confirmationPopup extends Popup {
+  constructor(popupSelector,  ) {
+    super({ popupSelector });
+  }
+  close(){
```

Comment:
You shouldn’t create the **same methods** of the parent class if you don’t change it inside and only call it with `super`. The parent methods are available in any inheritor of the class

**Please, correct it everywhere**.